src/relentless/model/potential/potential.py

```diff
@@ -375,3 +375,111 @@ def save(self, filename):
         data = self.to_json()
         with open(filename, "w") as f:
             json.dump(data, f, sort_keys=True, indent=4)
+
+
+class BondedPotential(Potential):
```

Comment:
Class docstring missing. It should document why this class exists for developers.